kalamine/tpl/base.ahk

```diff
@@ -61,10 +61,22 @@ SetTimer, ShowTrayTip, -1000  ; not working
 
 global DeadKey := ""
 
+UnicodeUpperIfCapsLock(unicode:="") {
+  CapsLockOn := GetKeyState("CapsLock", "T") 
+  if %CapsLockOn% {
```

Comment:
Out of curiosity: are these two lines actually needed because of some AHK dark magic ?

(Not requesting any change here, just asking.)

Reply:
Might be able to do `if % GetKeyState` I’ll test that tomorrow :)